corehq/apps/linked_domain/views.py

```diff
@@ -92,7 +92,7 @@ def brief_apps(request, domain):
 @login_or_api_key
 @require_linked_domain
 def released_app_versions(request, domain):
-    return JsonResponse({'versions': get_latest_released_app_versions_by_app_id(domain, include_remote=False)})
```

Comment:
This function only accepts one arg (and doesn't return remote results):
https://github.com/dimagi/commcare-hq/blob/d840b355701297c72b49e7db01523b35ef3ae06e/corehq/apps/app_manager/dbaccessors.py#L56-L56